install/kubernetes/cilium/values.yaml

```diff
@@ -1713,11 +1713,6 @@ resourceQuotas:
 # in the node.
 sleepAfterInit: false
 
-# -- Configure BPF socket operations configuration
-sockops:
-  # enabled enables installation of socket options acceleration.
-  enabled: false
-
```

Comment:
This file is generated from `install/kubernetes/cilium/values.yaml.tmpl`. Your change here is correct, but the right workflow would be to update that other template file, and then to run `make -C install/kubernetes` to update `values.yaml`. Can you please do that? It should alwo update `install/kubernetes/cilium/README.md` at the same time to remove the `sockops` mention there.